docs/resource-hsaccess.txt

```diff
@@ -0,0 +1,35 @@
+/home/igray/hydroshare/hs_modelinstance/models.py
```

Comment:
@ilangray This looks to me like your personal notes and I see your account name is included in the path as well. If so, this should not be included in the main hydroshare codebase repo and should be removed.